DominicGBauer/information-access
# feat: added search results component

# Pull Request

## Scope
https://dev.azure.com/JHNMAT015/ECO5037S%20Front%20end/_backlogs/backlog/ECO5037S%20Front%20end%20Team/Backlog%20items/?workitem=87
Added a stand-in/starting point for the search results once we start with the API. 

## Work Done
Created a component modeled off wireframe 2. 

## Steps to Test
```npm run start```
Should be visible if you navigate to /searchresults as shown below. 

## Screenshot
![image](https://user-images.githubusercontent.com/80746660/133227788-f6bae721-154f-4afc-9051-017dab82aa78.png)
